chromium_src/chrome/browser/chrome_resource_bundle_helper.cc

```diff
@@ -0,0 +1,16 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#define LoadLocalState LoadLocalState_ChromiumImpl
+#include "../../../../chrome/browser/chrome_resource_bundle_helper.cc"
+#undef LoadLocalState
+#include "brave/common/resource_bundle_helper.h"
+
+std::string LoadLocalState(
+    ChromeFeatureListCreator* chrome_feature_list_creator,
+    bool is_running_tests) {
+  std::string ret = LoadLocalState_ChromiumImpl(chrome_feature_list_creator, is_running_tests);
+  brave::InitializeResourceBundle();
+  return ret;
+}
```

Comment:
This is the location where Chromium initializes its resource bundles. I think a longer term better solution is for us to remove even this and instead repack with the chrome pak files.  

It was needed because we were initializing it here instead:
https://github.com/brave/brave-core/pull/1117#pullrequestreview-185818417

And this was happening too late after the commit referenced in the commit landed for Chromium.

More info for the change that was introduced that caused it here:
https://github.com/brave/brave-core/pull/1117/commits/a8297a23f240e672af01e482f0641fd1af42e5df